contracts/Pool.sol

```diff
@@ -76,8 +76,10 @@ abstract contract Pool {
     /// Whenever a proxy is added to a sender, this number is added to its `weightCount`.
     /// Limits costs of changes in sender's or proxy's configuration.
     uint32 public constant PROXY_WEIGHTS_COUNT_MAX = 10;
-    /// @notice The amount passed to `withdraw` to withdraw all the funds
+    /// @notice The amount passed as the withdrawn amount to withdraw all the funds
```

Comment:
Removing the “n” at the end aligns it with the parameter name (`withdrawAmt`) and makes it clearer that this controls the amount to be withdrawn and not the amount that has actually be withdrawn, that is the return value `withdrawn`.

```suggestion
    /// @notice The amount passed as the withdraw amount to withdraw all the funds
```

Reply:
Fixed